src/molecules/DowntimePage/index.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { render, screen } from '@testing-library/react';
+import DowntimePage from './index';
+
```

Comment:
@riya-2206 they are very basic test cases and are testing nothing from config.Plz import config and check if components are properly rendering based on the config.